.github/workflows/ruby.yml

```diff
@@ -27,11 +27,20 @@ jobs:
           - gemfiles/rails5_2.gemfile
           - gemfiles/rails6_0.gemfile
           - gemfiles/rails6_1.gemfile
+          - gemfiles/rails7_0.gemfile
         exclude:
+          - ruby-version: jruby
+            gemfile: gemfiles/rails7_0.gemfile
```

Comment:
rails 7.0 requires ruby >= 2.7, but jruby 9.3 is still compatible with ruby 2.6.